templates/azure/ebook/index.html

```diff
@@ -10,100 +10,72 @@
   https://docs.google.com/document/d/1JL_TeFBg1tSqiYIiXU0IwJTKWRDKJEXWtEgLRWFnToI/edit#
 {% endblock meta_copydoc %}
 
-{% block head_extra %}
-  <style>
-    h2,
-    .p-heading--2 {
-      font-weight: 300;
-    }
-  </style>
-{% endblock %}
+{% block body_class %}is-paper{% endblock body_class %}
 
 {% block content %}
-
-  <section class="p-strip--suru-bottomed">
-    <div class="row">
-      <div class="col-7 u-vertically-center">
-        <div>
-          <h1>Why do cloud innovators migrate to Ubuntu Pro?</h1>
-
-          <p class="u-no-padding--top p-heading--4">Get our eBook to find out</p>
-
-          <p>
-            <a href="#ubuntu-pro-ebook" class="p-button--positive">Download<span class="u-off-screen">the Why do Cloud innovators migrate to Pro? eBook by completing this form</span></a>
-          </p>
+<section class="p-section--hero">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="u-no-margin--bottom">Why do cloud innovators migrate to Ubuntu Pro?</h1>
+      <h2>Get our eBook to find out</h2>
+      <hr class="is-muted" />
+      <p>
+        <a href="#ubuntu-pro-ebook" class="p-button--positive">Download<span class="u-off-screen"> the Why do Cloud innovators migrate to Pro? eBook by completing this form</span></a>
```

Comment:
The download button should be below the image per design
<img width="887" alt="Screenshot 2024-08-19 at 2 50 56 PM" src="https://github.com/user-attachments/assets/1dfbe746-a77d-41f3-a10c-52939a2d075c">

Reply:
Agreed with Mattea that in our regular pattern the button is on the left

Reply:
Should the `hr` above the button be removed?
<img width="640" alt="Screenshot 2024-08-19 at 4 43 27 PM" src="https://github.com/user-attachments/assets/8570a850-0982-4be1-82ad-d3884da1952a">

I see that in the Figma design there were no buttons previously on the left side.
<img width="668" alt="Screenshot 2024-08-19 at 4 44 01 PM" src="https://github.com/user-attachments/assets/5fde0873-c9d3-459b-acfe-28876382a606">

Reply:
@mattea-turic can you pls confirm if the current design looks good to you?

Reply:
@britneywwc @lizzochek  it looks good to me :) hr was intentional